doc/index.rst

```diff
@@ -218,6 +218,38 @@ The following is a basic configuration of the controllers:
 .. literalinclude:: ../gazebo_ros2_control_demos/config/cart_controller.yaml
    :language: yaml
 
+
+Multiple Namespaces
+-----------------------------------------------------------
+The gazebo_ros2_control plugin can be launched in multiple namespaces. This is useful if it is desired to have multiple robots of the same description running at the same time.
+If the namespace is to be defined in the URDF file, then you can use the ``<namespace>`` tag as detailed below:
```

Comment:
It is possible to directly define the namespace in the urdf file, as showed below: